src/client/hud.cpp

```diff
@@ -54,14 +54,14 @@ Hud::Hud(Client *client, LocalPlayer *player,
 
 	tsrc = client->getTextureSource();
 
-	v3f crosshair_color = g_settings->getV3F("crosshair_color");
+	v3f crosshair_color = g_settings->getV3F("crosshair_color").value_or(v3f());
 	u32 cross_r = rangelim(myround(crosshair_color.X), 0, 255);
 	u32 cross_g = rangelim(myround(crosshair_color.Y), 0, 255);
 	u32 cross_b = rangelim(myround(crosshair_color.Z), 0, 255);
 	u32 cross_a = rangelim(g_settings->getS32("crosshair_alpha"), 0, 255);
 	crosshair_argb = video::SColor(cross_a, cross_r, cross_g, cross_b);
 
-	v3f selectionbox_color = g_settings->getV3F("selectionbox_color");
+	v3f selectionbox_color = g_settings->getV3F("selectionbox_color").value_or(v3f());
```

Comment:
sounds like you are a rust programmer

Reply:
i'd prefer we add a default setting in getV3F with a default= v3f()

Reply:
Optionals are perfectly fine for this

Reply:
> sounds like you are a rust programmer

I want to be one but not yet :sweat_smile:

Reply:
@appgurueu it's not a common C++ mode, generally we use defaults as it's permitted in this language :)

Reply:
we are here introducing a new pattern not present in our codebase, i'm not sure it's a good idea